src/language-server/converter.ts

```diff
@@ -0,0 +1,58 @@
+import * as path from 'path';
+import {Severity, SourcePosition, SourceRange, Warning} from 'polymer-analyzer';
+import {Diagnostic, DiagnosticSeverity, Position as LSPosition, Range} from 'vscode-languageserver';
+import Uri from 'vscode-uri';
+
+/**
+ * Converts between Analyzer and Editor Service types and LSP types.
+ */
+export class Converter {
+  constructor(private _workspaceUri: Uri) {
```

Comment:
Moved to a more explicit declaration of the field. This way we can also make it readonly.